lib/components/app/responsive-webapp.js

```diff
@@ -201,28 +204,20 @@ class RouterWrapper extends Component {
                 '/stop',
                 '/stop/:id'
               ]}
-              render={() => <WebappWithRouter {...this.props} />}
+              render={() => withLoggedInUser(<WebappWithRouter {...this.props} />)}
```

Comment:
Doesn't this need a little more nuance? Won't this just return the awaiting screen if no user is logged in?

Reply:
Good point on handling situations with auth0 disabled. What could be a better name? `includeLoggedInUser`?

Reply:
`supportLoggedInUser` it is.